pkg/azurelustre/nodeserver.go

```diff
@@ -357,3 +514,359 @@ func (d *Driver) ensureMountPoint(target string) (bool, error) {
 	}
 	return !notMnt, nil
 }
+
+func (d *Driver) createSubDir(vol *lustreVolume, subDirPath string, mountOptions []string) error {
+	if err := d.internalMount(vol, mountOptions); err != nil {
+		return err
+	}
+
+	defer func() {
+		if err := d.internalUnmount(vol); err != nil {
+			klog.Warningf("failed to unmount lustre server: %v", err.Error())
+		}
+	}()
+
+	internalVolumePath, err := getInternalVolumePath(d.workingMountDir, vol, subDirPath)
+	if err != nil {
+		return err
+	}
+
+	klog.V(2).Infof("Making subdirectory at %q", internalVolumePath)
+
+	if err := os.MkdirAll(internalVolumePath, 0775); err != nil {
```

Comment:
Will we want to allow for custom subdir perms down the road, or are we happy with read-only and this being the only options?

Reply:
This is a good point, I'll note it for consideration as a separate change